CHANGES/123.removal

```diff
@@ -0,0 +1 @@
+Moved the location of the `--version` command to the scope of the last command.
```

Comment:
Doesn't this change move `--repository` to the end?

Reply:
After the last command at least. There is no specific order of options.
How can i word it better?

Reply:
What about "Moved the location of \`--repository\` option after the last command"?